res/layout/file_actions.xml

```diff
@@ -3,12 +3,31 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="fill_parent"
     android:layout_height="fill_parent"
-    android:layout_margin="@dimen/standard_margin">
+    android:layout_margin="@dimen/standard_margin"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/file_actions_header"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/standard_padding"
+        android:textColor="@color/black"
```

Comment:
The default (don't know why, must be within app compat) is #6D6D6D:
![device-2015-09-10-142459](https://cloud.githubusercontent.com/assets/1315170/9788079/f694832c-57c7-11e5-9002-4ab49b815c5a.png)